packages/python-sdk/e2b/session/main.py

```diff
@@ -147,6 +154,47 @@ async def close(self) -> None:
         await super().close()
         await self._close()
 
+    def file_url(self) -> str:
+        """
+        Returns a URL that can be used to upload files to the session via a multipart/form-data POST request.
+        This is useful if you're uploading files directly from the browser.
+        The file will be uploaded to the user's home directory with the same name.
+        If a file with the same name already exists, it will be overwritten.
+        """
+        hostname = self.get_hostname(self._debug_port)
+        protocol = "http" if self._debug_dev_env == "local" else "https"
+        return f"{protocol}://{ENVD_PORT}-{hostname}/file"
+
+    def upload_file(self, file: BufferedReader) -> str:
```

Comment:
you want probably `IO` from `typing` module. This will light up in the IDE if you want to use `open(...)`
```suggestion
    def upload_file(self, file: IO) -> str:
```